apis/feature_extraction/inference/app/feature_extraction.py

```diff
@@ -0,0 +1,24 @@
+import torch
+from torchvision import transforms
+from torchvision.models.vgg import vgg16
+
+device = (
+    torch.device('cuda')
+    if torch.cuda.is_available()
+    else torch.device('cpu')
+)
+
+transform_pipeline = transforms.Compose([
+    transforms.Resize((224, 224)),
+    transforms.ToTensor(),
+    transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
```

Comment:
They're magic, but they're well known magic in ML - they're the mean and stdev of the Imagenet dataset which is used to train VGG. If we were to fine-tune VGG on our own data, these might change, but I don't think we'll be in that situation any time soon. Happy to defer if you feel strongly, but imo tucking things like this away into config files or separate variables just obfuscates the magic, rather than getting rid of it.

Reply:
Got it, in that case a conf file doesn't make sense but I don't think separate variables obfuscate the magic at all - something like `imagenet_mean` and `imagenet_stddev`?